packages/wonder-blocks-data/components/data.test.js

```diff
@@ -748,6 +778,37 @@ describe("Data", () => {
                     ).not.toHaveBeenCalled();
                 });
 
+                it("should defer to handler if interceptor shouldRefreshCache returns null", () => {
+                    // Arrange
+                    const fakeHandler: IRequestHandler<string, string> = {
+                        fulfillRequest: jest.fn(() => Promise.resolve("data")),
+                        getKey: (o) => o,
+                        shouldRefreshCache: jest.fn(() => false),
+                        type: "MY_HANDLER",
+                        cache: null,
+                    };
+                    const fakeChildrenFn = jest.fn(() => null);
+                    const shouldRefreshCacheFn = jest.fn(() => null);
```

Comment:
I was like why would we want to return `?boolean` but it's so that we can defer to Data's `shouldRefreshCacheFn`, neat!